bower.json

```diff
@@ -16,7 +16,7 @@
     "iron-dropdown": "PolymerElements/iron-dropdown#^2.1.0",
     "paper-icon-button": "PolymerElements/paper-icon-button#^2.0.1",
     "paper-styles": "PolymerElements/paper-styles#^2.0.0",
-    "etools-behaviors": "d0c60e9"
+    "etools-behaviors": "89f132c"
```

Comment:
I made some updated to etools packages to fix `appliesMixin` annotation...  Please update versions according with the migration document :)